readme.md

```diff
@@ -25,7 +26,11 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```
+To submit a form using AJAX, we need to
+1. capture the form submit button
+2. get all the data from the form using jQuery
+3.Submit using AJAX (we can use the $.post() method)
```

Comment:
:+1:

Although we haven't used plain ol' `$.post` in class either. In this case, you should use `$.ajax`.